locations/spiders/societe_generale.py

```diff
@@ -8,41 +10,47 @@
 class SocieteGeneraleSpider(scrapy.Spider):
     name = "societe_generale"
     item_attributes = {"brand": "Societe Generale", "brand_wikidata": "Q270363"}
-    allowed_domains = ["societegenerale.com"]
+    allowed_domains = ["societegenerale.com", "agences.societegenerale.fr"]
     start_urls = [
-        "https://www.societegenerale.com/en/about-us/our-businesses/our-locations",
+        "https://agences.societegenerale.fr/banque-assurance/sitemap_agence_pois.xml",
     ]
 
     def parse(self, response):
```

Comment:
You can inherit a [SitemapSpider](https://docs.scrapy.org/en/latest/topics/spiders.html#sitemapspider) then you don't have to do this function.

Reply:
I tried using sitemap_urls (with and without sitemap_rules), but for some reason it doesn't parse anything. Could it be due to the sitemap URL taking a bit to load? https://agences.societegenerale.fr/banque-assurance/sitemap_agence_pois.xml

```
sitemap_urls = (
        "https://agences.societegenerale.fr/banque-assurance/sitemap_agence_pois.xml",
    )
sitemap_rules = [(r"https://agences.societegenerale.fr/banque-assurance/.*", "parse")]

```

`2022-06-30 10:58:11 [scrapy.extensions.logstats] INFO: Crawled 0 pages (at 0 pages/min), scraped 0 items (at 0 items/min)`

Reply:
See https://github.com/alltheplaces/alltheplaces/pull/3709/commits/267766283fb41a3a2697a7772e96c94043f21bb5.

I don't know exactly what you did, but maybe you didn't rename `parse_store` to `parse`?
By default everything found in the sitemap is passed to `parse`,